libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1654,14 +1656,6 @@ void AP_GPS::calc_blended_state(void)
         _hgt_offset_cm[i] = (float)(state[GPS_BLENDED_INSTANCE].location.alt - state[i].location.alt) *  alpha[i] + _hgt_offset_cm[i] * (1.0f - alpha[i]);
     }
 
-    // Calculate a corrected location for each GPS
```

Comment:
If we are going to remove this then the calculation of `_NE_pos_offset_m` and `_hgt_off_cm` above should also be removed. Also worth observing they should have been locals here, rather then members of the GPS object. (Given that we track `_blended_antenna_offset` directly I think removing this is fine, but lets remove all the dead bits of code.

Reply:
done, note that this ended up removing quite a bit of code